.github/workflows/dh-ci-frontend.yml

```diff
@@ -13,11 +13,10 @@
 # limitations under the License.
 
 # DataHub UI Frontend and BFF
-name: DH CI
+name: DH CI frontend
 
-concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: true
```

Comment:
so concurrency isn't needed anymore? Should the new orchestrator have this instead?